x-pack/qa/security-migrate-tests/build.gradle

```diff
@@ -30,6 +30,7 @@ integTestCluster {
             retries: 10)
     return tmpFile.exists()
   }
+  // TODO: systemProperty('tests.cluster', "${-> cluster.transportPortURI }") when migerating to testclusters
```

Comment:
Why is this commented out?

Reply:
because it's just a reminder for when we convert it to testclusters, it doesn't use it yet, and we continue to set that property for ClusterFormationTasks